docs/elements.md

```diff
@@ -413,20 +413,21 @@ Element to arrange given blocks of code or text that are displayed initially in
+| `max-incorrect`       | integer                 | special                        | The maximum number of incorrect answers to be displayed in the source area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.                                                                                                                                                                                              |
+| `min-incorrect`       | integer                 | special                        | The minimum number of incorrect answers to be displayed in the source area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.                                                                                                                                                                                              |
+| `source-header`       | string                  | "Drag from here"               | The text that appears at the start of the source area.                                                                                                                                                                                                                                                                                                                                    |
+| `solution-header`     | string                  | "Construct your solution here" | The text that appears at the start of the solution area.                                                                                                                                                                                                                                                                                                                                  |
+| `solution-placement`  | "right" or "bottom"     | "right"                        | `right` shows the source and solution areas aligned side-by-side. `bottom` shows the solution area below the source area.                                                                                                                                                                                                                                                                 |
+| `partial-credit`      | string                  | Depends on `grading-method`    | For the `dag` grading method, you may specify `none` for no partial credit (defualt) or `lcs` for partial credit based on the LCS edit-distance from the student solution to some correct solution. For the other grading methods, using this property is not yet supported. Grading methods `unordered` and `ranking` will assign partial credit, and grading method `ordered` will not. |
```

Comment:
Edit for possibly increased clarity:
```suggestion
| `partial-credit`      | string                  | Depends on `grading-method`    | For the `dag` grading method, you may specify `none` for no partial credit (defualt) or `lcs` for partial credit based on the LCS edit-distance from the student solution to some correct solution. For the other grading methods, using this property is not yet supported. Grading methods `unordered` and `ranking` will always assign partial credit, and grading method `ordered` will never do so. |
```
Also, instead of `string` for the type, let's enumerate the possible values like the other properties here do.

Reply:
ok